account_partner_required/readme/CONFIGURE.rst

```diff
@@ -0,0 +1,6 @@
+[ This file is optional, it should explain how to configure
```

Comment:
Please remove this text

Reply:
@rven requested change to approve into your fork https://github.com/rven/account-financial-tools/pull/4